frontend/src/components/Notification.jsx

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from "react";
+import Avatar from "./Avatar.jsx";
+
+//images
+import avatar_image from "../images/placeholder-images/avatar.png";
+
+class Notification extends Component{
+
+    render(){
+        return(
+            <div className="notification">
+                <div className="notification__profile-wrap">
+                    <Avatar avatarSrc={avatar_image}/>
+                    <h3 className="notification__username">@Shwetal </h3>
+                    <p className="notification__article">added this article</p>
+                </div>
+                <div className="notification__review-wrap">
+                    <a href="#" className="button__xsmall button__xsmall--hollow">Review</a>
```

Comment:
In the design, you have used the small variant of the button. Was there any reason you had to make xsmall variant? 🤔

Reply:
@salil-naik 
as per figma design i thought that the buttons existing in _button.scss won't fit here. they were too big for notification component.....So, had to create that xsmall variant

Reply:
Use `button__small button__small--hollow`